tests/acceptance/book/CreateBookCest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use \Codeception\Util\HttpCode;
+
+class CreateBookCest
+{
+
+	public function _before(AcceptanceTester $I)
+	{
+		$I->loginAsAdmin();
+	}
+
+	/**
+	 * @example { "blogPublic": "1" }
+	 * @example { "blogPublic": "0" }
+	 */
+	public function tryToCreateABook(AcceptanceTester $I, \Page\Acceptance\CreateBook $createBookPage, \Codeception\Example $example)
+	{
+    	$bookWebAddress = "book".rand();
+    	$bookTitle = "$bookWebAddress Title";
+		$createBookPage->createBook($bookWebAddress, $bookTitle, boolval( $example['blogPublic'] ) );
```

Comment:
we could use casting `(bool) $example['blogPublic']` instead of `boolval` function.